src/ProfileEndpoints.jl

```diff
@@ -208,7 +209,8 @@ function allocations_start_endpoint(req::HTTP.Request)
 end
 
 function allocations_stop_endpoint(req::HTTP.Request)
-    return _stop_alloc_profile()
+    # Defer the potentially expensive profile symbolication to a non-interactive thread
+    return fetch(Threads.@spawn _stop_alloc_profile())
```

Comment:
oh yes that seems wise! 💡 

I suppose we could even spell this as:
```suggestion
    return fetch(Threads.@spawn :default _stop_alloc_profile())
```
just so that code is even more explicit

(but maybe that's overkill since its the usual behaviour of `spawn` and there's the comment, i just sometimes forget whether or not `spawn` inherits the threadpool)

Reply:
Thanks Nick. I started with that but then I noticed the docs say that `(:default if [threadpool is] unspecified)` and we also need to support Julia 1.6 so it felt a bit cleaner to leave as is.